deploy/dex_structure.py

```diff
@@ -1944,3 +1948,27 @@ def metastaking_deploy(self, contracts_index: str, deployer_account: Account, ne
 
             deployed_contracts.append(deployed_metastaking_contract)
         self.contracts[contracts_index].deployed_contracts = deployed_contracts
+
+    def position_creator_deploy(self, contracts_index: str, deployer_account: Account, network_providers: NetworkProviders):
+        contract_structure = self.contracts[contracts_index]
+        deployed_contracts = []
+        for contract_config in contract_structure.deploy_structure_list:
+            position_creator_contract = PositionCreatorContract(
+                "", contract_config['egld_wrapped_address'], contract_config['wrapped_token_id']
```

Comment:
This is ok, but we will probably change it later on to use the deployed wrapper contracts (as well).
This is something interesting to investigate later on, cause it might be useful to have both cases: explicit definition of the linked contract or usage of already deployed one (available in the deploy structure).

Reply:
Usage of existing wrappers are a bit problematic for the moment due to their nature of being deployed in different shards. Saving them in the deploy structure is done manually right now since there's no support for deploy from multiple addresses.